### PR TITLE
Implemented the NeuroGameRenderer class

### LINKED ISSUE #174: Neuro-Visual Feedback Module should be implemented
**Actual Behaviour**
* Currently the application lacks the implementation of the Neuro-Visual Feedback Module which includes the classes - FocusOMeter, FocusOMeterGLRenderer, LongtermFFTVisualizer, NeuroGameRenderer, NeuroGameVisuals, OSCFeedback, ZenSpaceGLRenderer, ZenSpaceVisuals.

**Expected Behaviour**
* All these classes should be correctly ported after converting them to Android Version.
* The usages of Jogamp should be appropriately substituted everywhere with the correct replacement for Android Optimization.

**Classes to Port and Checklist of the work** 
- [x] FocusOMeter
- [x] FocusOMeterGLRenderer
- [x] LongtermFFTVisualizer
- [x] NeuroGameRenderer
- [x] NeuroGameVisuals
- [x] OSCFeedback
- [x] ZenSpaceGLRenderer
- [x] ZenSpaceVisuals

**Would you like to work on the issue?**
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #174 partly (Checklist point - 4)

**Changes**: 
* Implemented the class NeuroGameRenderer (belonging to the output.visual package) in the Android version.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@mariobehling @CloudyPadmal Could you please review the PR?

**Review by CloudyPadmal — APPROVED** [review]

**Comment by suresh-gandhi** [comment]
@CloudyPadmal Does something needs to be changed or it can be merged?

File: app/src/main/java/io/neurolab/main/output/visual/NeuroGameRenderer.java [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+        this.height = height;
+
+        if (texYantra == null) {
+            texYantra = generateWhiteNoise(gl2, 800, 800);
+        }
+
+        if (width < height) {
+            shortSide = width;
+            offsetX = 0;
+            offsetY = height / 2;
```
Comment (jddeep) [inline app/src/main/java/io/neurolab/main/output/visual/NeuroGameRenderer.java:100]: @suresh-gandhi I think these are again UI related. Should be done when we code the Fragments.

**Review by jddeep — CHANGES_REQUESTED** [review]

**Review by jddeep — COMMENTED** [review]
@suresh-gandhi Have a revision of the whole class and remove all UI related actions.

**Comment by suresh-gandhi** [comment]
@jddeep You are correct about the UI and base functionality separation but these classes are rendering classes (using the open GL library) themselves as you can see. Thanks for your review.

**Comment by suresh-gandhi** [comment]
@CloudyPadmal @mariobehling Could you please review and merge the PR.

**Comment by jddeep** [comment]
> @jddeep You are correct about the UI and base functionality separation but these classes are rendering classes (using the open GL library) themselves as you can see. Thanks for your review.

Alright @suresh-gandhi we can keep it for now. Will update accordingly when we start working on UX/UI.

**Review by jddeep — APPROVED** [review]

**Pull request merged**